### DISCUSSION TITLE
Hỏi bài nhau vô đây hỏi nhe mng

### DISCUSSION THREAD

**Discussion by LeBinhMinh10a6** [LeBinhMinh10a6/Nh-m-4---10C6-IT#6] [General]
Ai hok hiểu j dzô đây hỏi bài nhe

**Comment by PhuongTamNguyen** [comment]
câu 3 bài 8 là như nèo á, tui hong hiểu lắm

**Reply by aimabietduoc** [reply]
đợi tui xí đang đi hỏi kkkk

**Comment by PhuongTamNguyen** [comment]
uki lunnnnnnn

**Comment by aimabietduoc** [comment]
tui đi hỏi bạn đó kiu là tạo ra cái trang web gì đó thui là đc hay sao á, tạo web gì cũng đc hay sao á

**Comment by PhuongTamNguyen** [comment]
tạo web gì choa hay haaaa

**Comment by aimabietduoc** [comment]
tui cũn k bit nhma thấy bảo web gì cũng đc á

Tải Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Từ: PhuongTamNguyen ***@***.***>
Đã gửi: Thursday, November 18, 2021 9:21:32 PM
Đến: LeBinhMinh10a6/Nh-m-4---10C6-IT ***@***.***>
Cc: Nguyễn Thế Khôi ***@***.***>; Comment ***@***.***>
Chủ đề: Re: [LeBinhMinh10a6/Nh-m-4---10C6-IT] Hỏi bài nhau vô đây hỏi nhe mng (Discussion #6)


tạo web gì choa hay haaaa

**Comment by BaoNguyen30** [comment]
Mình có cần trang trí hong mng

**Comment by PhuongTamNguyen** [comment]
nếu trang trí thì mình làm canva hay dìi
